src/contracts/pods/EigenPodManager.sol

```diff
@@ -675,12 +666,7 @@ contract EigenPodManager is
             shareAmounts[0] = podOwnerShares[podOwner];
             IStrategy[] memory strategies = new IStrategy[](1);
             strategies[0] = beaconChainETHStrategy;
-            delegationManager.decreaseDelegatedShares(podOwner, strategies, shareAmounts);
-
-            // undelegate the pod owner, if possible
-            if (stakerCanUndelegate(podOwner)) {
-                _undelegate(podOwner);
-            }
+            delegationManager.decreaseDelegatedShares(podOwner, strategies, shareAmounts, true/*undelegateIfPossible*/);
         }
```

Comment:
inline comment is mistake?

Reply:
are you talking about this comment?
```
        // if the podOwner has delegated shares, record a decrease in their delegated shares
```
I think this is correct? Maybe a bit unclear though; would appreciate any suggestions you have for replacement.